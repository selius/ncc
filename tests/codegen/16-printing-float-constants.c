int main()
{
	__print_float(0.27);
	__print_float(12.34);
	return 0;
}
