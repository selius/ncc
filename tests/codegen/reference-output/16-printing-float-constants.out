0.270000
12.340000
